app/domain/errors.rb

```diff
@@ -519,6 +519,27 @@ module AuthnJwt
         msg: "mandatory claims value '{0-mandatory-claims-value}' should not contains claim name with spaces.",
         code: "CONJ00109E"
       )
+
+      MappedClaimsMissingInput = ::Util::TrackableErrorClass.new(
+        msg: "Failed to parse mapped claims, mapped claims value is empty, or was not found.",
```

Comment:
I feel like we are repeating the same issue twice.
maybe it should say:
Failed to parse mapped claims, the value is empty or was not found